css/index.css

```diff
@@ -1,3 +1,311 @@
+.container {
+	width: 880px;
+	/* TB LR */
+	margin: 0 auto;
+  }
+  
+  /* header */
+  
+  header nav {
+	display: flex;
+	width: 691px;
+	margin-top: 35px;
+	justify-content: space-between;
+  }
+  
+  header nav a {
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+  
+  header nav a:hover {
+	text-decoration: underline;
+  }
+  header {
+	  display:flex;
+	  justify-content: space-between;
+	  width: 920px;
+
+  }
+  header .logo {
+	display: inline-block;
+	margin-top: 25px;
+  }
+  
+  /* cta */
+  .cta{
+	  width: 920px;
+	  display: flex;
+	  justify-content: space-around;
+  }
+  .cta .cta-text {
+	display: inline-block;
+	vertical-align: top;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+  }
+  
+  .cta .cta-text button {
+	background: white;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 8px 44px;
+	font-size: 17px;
+	border: 1px solid black;
+	cursor: pointer;
+  }
+  
+  .cta .cta-text button:hover {
+	background: black;
+	color: white;
+  }
+  
+  .cta img {
+	display: inline-block;
+	margin-top: 69px;
+  }
+  
+  .main-content {
+	border-top: 1px solid black;
+	margin-top: 76px;
+	padding-top: 39px;
+  }
+  
+  .main-content .top-content {
+	display:flex;
+	justify-content: space-between;
+
+
+  }
+  
+  .main-content .top-content .text-content {
+	width: 410px;
+
+  }
+  
+  .main-content .middle-img {
+	margin-top: 45px;
+  }
+  
+  .main-content .bottom-content {
+	margin-top: 37px;
+	display:flex;
+	justify-content: space-between;
+  }
+  
+  .main-content .bottom-content .text-content {
+	width: 275px;
+  }
+  
+  .contact {
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+  }
+  
+  .contact p {
+	margin: 20px 0;
+  }
+  
+  footer {
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+  }
+/* Media queries go here */
+@media only screen and (max-width: 800px) {  
+    .cta img {
+        display: none;
+      }
+    header nav{
+        width: 800px;
+        justify-content: space-around;
+    
+    }
+    header{
+        display: flex;
+        flex-direction: column;
+        width: 800px;
+    }
+    header .logo{
+        order: -1;
+        align-self: center;
+    }
+    .main-content .top-content .text-content {
+        width: 360px;
+    
+      }
+    .main-content{
+        width: 800px;
+    }
+    .main-content .top-content{
+        justify-content: space-around;
+        width: 800px;
+    }
+    .middle-img{
+        width: 760px;
+        height: auto;
+        margin-right: 20px;
+        margin-left:20px;
+    }
+    .container{
+        width: 800px;
+    }
+    .main-content .bottom-content .text-content {
+        width: 210px;
+       
+    }
+    .main-content .bottom-content{
+        width: 800px;
+        justify-content: space-around;
+    }
+    .contact{
+        width: 800px;
+        padding-left: 20px;
+    }
+}
+@media only screen and (max-width: 500px) {  
```

Comment:
For some reason there's a white space on the side when I look at this in mobile view. I think maybe you can try just using @media (max-width: 500px)?